packages/common/src/types/projects.ts

```diff
@@ -178,7 +178,7 @@ export type CreateSnowflakeCredentials = {
     accessUrl?: string;
     startOfWeek?: WeekDay | null;
     quotedIdentifiersIgnoreCase?: boolean;
-    override?: string;
```

Comment:
what is this supposed to do?

Reply:
it was actually a boolean but typed as string for some reason